### ISSUE TITLE
Product Button: ensure adding to cart is announced to screen readers

### ISSUE THREAD

**Issue by Aljullu** [woothemes/woocommerce#59438]
When a product is added to cart via the *Add to Cart + Options* or *Product Collection* blocks, it should be announced to users using screen readers.

## **To Reproduce**

1. Go to *Appearance* > *Editor* > *Templates* > *Single Product*.
2. Click on the *Add to Cart with Options* block and update to the blockified version.
3. In the frontend, using a screen reader, add a product to the cart.
4. Notice nothing is announced.

## **Expected behavior**

There should be an announcement that a product has been added to the cart.
